samples/httpd_bsd/main.c

```diff
@@ -0,0 +1,107 @@
+#include "lwip/debug.h"
```

Comment:
nit: As above, `#include <lwip/debug.h>` should be preferred here.

/edit: Eh, seems like this comes from the httpd sample. We should probably just clean up all occurrences like these in a follow-up PR.